ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -337,10 +458,42 @@ private class JdbcLedgerDao(
       offsetStep: OffsetStep,
       transaction: CommittedTransaction,
       divulged: Iterable[DivulgedContract],
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
-    throw new UnsupportedOperationException(
-      "not supported by append-only code"
-    ) // TODO append-only: cleanup
+      workflowId: Option[String],
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+
+    val builder = RawDBBatchPostgreSQLV1.Builder()
+    val events = transactionAcceptedToEvents(
```

Comment:
I'd be more fine like this instead of artificially constructing that struct in the JdbcLedgerDao. But I'll adapt nevertheless so it doesn't impact the `UpdateToDBDTOV1`

Reply:
Actually, `UpdateToDBDTOV1.apply` also needs the `participantId` (which is not used for `storeTransaction`), which is not available at that call-site. Wondering which change is more lenient to the codebase. What do you think @rautenrieth-da ?